packages/spanner/src/database.js

```diff
@@ -892,6 +892,11 @@ Database.prototype.runTransaction = function(options, runFn) {
 
   options = extend({}, options);
 
+  this.debug([
+    'Starting a transaction. Note that nested transactions are not currently',
```

Comment:
Let us remove "currently" from this. "currently" indicates that they might be supported at some point in time.